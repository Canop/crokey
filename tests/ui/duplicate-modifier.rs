fn main() {
    crokey::key!(ctrl-ctrl-5);
    crokey::key!(alt-alt-5);
    crokey::key!(shift-shift-5);
    crokey::key!(shift-alt-shift-ctrl-5);
}
