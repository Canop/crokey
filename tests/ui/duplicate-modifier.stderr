error: duplicate modifier ctrl
 --> tests/ui/duplicate-modifier.rs:2:23
  |
2 |     crokey::key!(ctrl-ctrl-5);
  |                       ^^^^

error: duplicate modifier alt
 --> tests/ui/duplicate-modifier.rs:3:22
  |
3 |     crokey::key!(alt-alt-5);
  |                      ^^^

error: duplicate modifier shift
 --> tests/ui/duplicate-modifier.rs:4:24
  |
4 |     crokey::key!(shift-shift-5);
  |                        ^^^^^

error: duplicate modifier shift
 --> tests/ui/duplicate-modifier.rs:5:28
  |
5 |     crokey::key!(shift-alt-shift-ctrl-5);
  |                            ^^^^^
